querying/ecosystems.go

```diff
@@ -0,0 +1,20 @@
+package querying
+
+type FindingEcosystemType uint8
+
+const (
+	FindingEcosystemApt FindingEcosystemType = iota
```

Comment:
I'm torn on keeping these as `iota` or making them strings. The real potential problem I see with `iota` is if we ever persist data from `vulnbot` into an external system (such as Datadog or Elastic), then the order of these values _can never change_, otherwise we will corrupt the data.

Open to thoughts here.

Reply:
Is there a downside to making them `string`? 

+1 feels like relying on order / implicit ints for enums could lead to a nasty bug down the road

Reply:
Downside... gotta manually write all of them out? lol. Not really much else that I can think of.